Mathlib/MeasureTheory/Integral/Average.lean

```diff
@@ -211,4 +223,215 @@ theorem set_average_const {s : Set α} (hs₀ : μ s ≠ 0) (hs : μ s ≠ ∞)
     ENNReal.one_toReal, one_smul]
 #align measure_theory.set_average_const MeasureTheory.set_average_const
 
+-- porting note: was `@[simp]` but `simp` can prove it
+theorem integral_average (μ : Measure α) [IsFiniteMeasure μ] (f : α → E) :
+    ∫ _, ⨍ a, f a ∂μ ∂μ = ∫ x, f x ∂μ := by
+  simp
```

Comment:
```suggestion
    ∫ _, ⨍ a, f a ∂μ ∂μ = ∫ x, f x ∂μ := by simp
```
no?

Reply:
Feel free to push to this branch. It is porting your code.